src/data_profiles.jl

```diff
@@ -35,56 +35,46 @@ function data_ratios(H :: Array{Float64,3}, N :: Vector{Float64}; τ :: Float64=
   end
 
   # Replace all NaNs with twice the max ratio and sort.
-  max_data = maximum(T)
+  max_data = maximum(T[.!(isnan.(T))])
   T[isnan.(T)] .= 2 * max_data
   T .= sort(T, dims=1)
   return (T, max_data)
 end
 
-data_ratios(H :: Array{TH,3}, N :: Vector{TN}; τ :: Tt=convert(Tt, 1.0e-3)) where {TH <: Number, TN <: Number, Tt <: Number} = data_ratios(convert(Array{Float64,3}, H), convert(Vector{Float64}, N), τ=convert(Float64, τ))
+function data_profile_axis_labels(labels, ns, operations, τ; kwargs...)
+  length(labels) == 0 && (labels = [@sprintf("column %d", col) for col = 1 : ns])
```

Comment:
```suggestion
  length(labels) == 0 && (labels = ["column $col" for col = 1 : ns])
```
Since there's no formatting.